core/ui/src/main/kotlin/de/tum/informatics/www1/artemis/native_app/core/ui/exercise/ExerciseListItem.kt

```diff
@@ -79,16 +110,40 @@ private fun ExerciseTypeIcon(modifier: Modifier, exercise: Exercise) {
     Box(modifier = modifier) {
         Icon(
             modifier = Modifier
-                .fillMaxSize()
-                .padding(8.dp),
+                .fillMaxSize(),
             painter = getExerciseTypeIconPainter(exercise),
             contentDescription = null
         )
     }
 }
 
 /**
- * Displays the exercise title, the due data and the participation info. The participation info is automatically updated.
+ * Displays a rectangle next to the text to show the difficulty of the exercise.
+ */
+@Composable
+private fun DifficultyRectangle(difficulty: Exercise.Difficulty) {
```

Comment:
This composable should have a modifier parameter to keep the code style consistent among the project.